schema.graphql

```diff
@@ -1,27 +1,30 @@
 type Contribution @entity {
   id: ID!
-
+  extrinsicHash: String! @index
   paraId: Int! @index
-  vaultId: String!
+  vault: Vaults! @index
   blockHeight: Int! @index
   account: String! @index
   amount: String!
-  referralCode: String
   timestamp: Date!
 }
 
 type Vaults @entity {
   id: ID!
-
-  createdAt: String
+  createdAt: Int
   paraId: Int! @index
-  vaultId: String!
   ctokenId: Int!
   phase: String!
   contributions: Int!
   totalAmount: String!
   contributionStrategy: String!
   cap: String!
   endBlock: Int!
-  trieIndex: Int!
+}
+
+type DissolvedVault @entity {
```

Comment:
Should we add totalAmount and contributions to dissolvedVault for debug easily in the future?